osu.ElasticIndexer/IndexQueueProcessor.cs

```diff
@@ -90,6 +90,7 @@ private void handleResponse(BulkResponse response, IEnumerable<ScoreItem> items)
                 }
 
                 stop();
+                return;
```

Comment:
Seems left over / unnecessary.

Reply:
It prevents the code after it from running as item statuses and errors are not set completely when `!response.isValid`

Reply:
Aha, I was at the wrong indentation level. The `stop()` call is a bit weird but seems okay for now.